cardinal/server/server_test.go

```diff
@@ -610,7 +610,7 @@ func TestCanCreateAndVerifyPersonaSigner(t *testing.T) {
 	assert.NilError(t, err)
 }
 
-func TestSigVerificationChecksNamespace(t *testing.T) {
+func TestSigVerificationChecksNamespaceAndSignature(t *testing.T) {
```

Comment:
this test is getting pretty long, might be nice to convert this to a table-driven test

Reply:
done